server/sockets/list.js

```diff
@@ -394,14 +410,18 @@ const changeListType = (socket, dashboardClients, cohortClients, listClients) =>
                 );
 
                 if (listClients.has(userId)) {
-                  socket.broadcast
-                    .to(listClients.get(userId))
-                    .emit(ListActionTypes.LEAVE_ON_TYPE_CHANGE_SUCCESS, {
-                      cohortId,
-                      isCohortMember,
-                      listId,
-                      type
-                    });
+                  const { viewId, socketId } = listClients.get(userId);
+
+                  if (viewId === listId) {
+                    socket.broadcast
+                      .to(socketId)
+                      .emit(ListActionTypes.LEAVE_ON_TYPE_CHANGE_SUCCESS, {
+                        cohortId,
+                        isCohortMember,
+                        listId,
+                        type
+                      });
+                  }
```

Comment:
the same here

Reply:
we have to first check if exist then destructure to certain props as we discussed